src/app/app-routing.module.ts

```diff
@@ -4,14 +4,16 @@ import { UserlistComponent } from './userlist/userlist.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
 import { CaseComponent } from './case/case.component';
 import { AssignedCasesComponent } from './AssignedCases/AssignedCases.component';
-
+import { config } from 'rxjs';
+import { PersonProfileComponent } from './person-profile/person-profile.component';
 
 const routes: Routes = [
   { path: '', redirectTo: '/dashboard', pathMatch: 'full' },
   { path: 'dashboard', component: DashboardComponent },
   { path: 'userlist', component: UserlistComponent },
-  { path: 'case', component: CaseComponent },
-  { path: 'AssignedCases', component: AssignedCasesComponent }
+  { path: 'positive-case', component: CaseComponent },
```

Comment:
I can change the naming of this component to positive case, I think it will be easier in my IDE. I would want to do it quick so that it doesn't cause merge conflicts with anyone else.

Reply:
If you wish to do that, I would advise making a new branch off Jake's committing it to that and PR-ing into this one. Since this is Jake's branch there can be conflicts much worse than merge conflicts when two people commit to the same branch. (Sounds stupid and tedious, but I got bit by this a lot in my time at Rose and in my professional career. I coined this kind of conflict as "Eating commits", where all git history is destroyed and both people lose key info)